lib/intercom-rails/script_tag.rb

```diff
@@ -121,6 +123,17 @@ def user_hash
       OpenSSL::HMAC.hexdigest("sha256", secret, (user_details[:user_id] || user_details[:email]).to_s)
     end
 
+    def session_duration_from_config
+      session_duration = IntercomRails.config.session_duration
+      if session_duration && valid_session_duration?(session_duration)
+        session_duration
+      end
+    end
+
+    def valid_session_duration?(session_duration)
+      session_duration.is_a?(Integer) && session_duration > 0
```

Comment:
The documentation says the minimum authorised time is 5 mins yet here we allow any time > 0. Should we check that it's at least 5 mins here?